parachain/pallets/ethereum-beacon-client/src/benchmarking/data.rs

```diff
@@ -64,7 +70,8 @@ pub fn initial_sync<SyncCommitteeSize: Get<u32>, ProofSize: Get<u32>>(
                 hex!("d33a17a3903ceac967c0afc2be32962dd69f5836e7674b4c30b2c68116720b2c").into(),
                 hex!("0d0607530d6ffd3dfffafee157c34db1430cd7a1f29dea854769cf5c45aed99d").into()
             ].try_into().expect("too many branch proof items"),
-            validators_root: hex!("270d43e74ce340de4bca2b1936beca0f4f5408d9e78aec4850920baf659d5b69").into()
+            validators_root: hex!("270d43e74ce340de4bca2b1936beca0f4f5408d9e78aec4850920baf659d5b69").into(),
+            import_time: time_now + 97200,
```

Comment:
Let's mention that this is the 27 hour sync committee period:
```suggestion
            import_time: time_now + 97200, // now + 27 hour sync committee period
```